### DISCUSSION TITLE
Score Output Interpretations

### DISCUSSION THREAD

**Discussion by sbpkyle** [chaidiscovery/chai-lab#70] [Q&A] [answered]
Hi, I had some protein-ligand complex predictions, and was trying to analysis it's output. However, I was unfamiliar with the 'aggregate_score'? 

I wanted some clarification on this, and possibly the other such as 'has_inter_chain_clashes', 'chain_intra_clashes', 'chain_chain_inter_clashes'.

Thank you in advanced!

**Comment by arogozhnikov** [comment, marked as answer]
HI @sbpkyle 

aggregate score is defined here:

https://github.com/chaidiscovery/chai-lab/blob/1c9ddb35f1fbbba3f106272ab5a155bfb7010445/chai_lab/ranking/rank.py#L95C1-L99C6

different chain clashes are how many atoms are too close: intra - clashed atoms within same chain, inter - from different chains.
I've simplified this recently to just provide a matrix which includes both inter and intra.

has_inter_chain_clashes - self-descriptive. We use it to assign low aggregate score to candidates with inter-chain clashes

**Reply by sbpkyle** [reply]
Hi thank you for the timely response, so just by looking at the definition for the aggregate score could you possibly generalize it as an overall confidence score of the prediction (i.e b/c it's accounting for multiple metrics)?

**Reply by arogozhnikov** [reply]
I don't understand your question. If you want to use some other metric, you surely can do that. That's just the one we use by default in comparisons and on the server to rank candidates

**Reply by sbpkyle** [reply]
Ah no worries! To clarify a bit, at least from my understanding, the aggregate score that is generated is used to rank the outputted predictions (i.e higher score = higher ranking).  This means for a predicted structure with a high aggregate score (I'm assuming closest to 1), that the model is confident that this structure is representative of the 'true' structure.

**Reply by arogozhnikov** [reply]
yes, that's the idea
